src/sinks/influxdb/logs.rs

```diff
@@ -202,11 +206,17 @@ impl HttpEventEncoder<BytesMut> for InfluxDbLogsEncoder {
         };
 
         // Timestamp
-        let timestamp = encode_timestamp(match log.remove(log_schema().timestamp_key()) {
+        let timestamp = encode_timestamp(match log.remove_timestamp() {
             Some(Value::Timestamp(ts)) => Some(ts),
             _ => None,
         });
 
+        // Add the `host` and `source_type` to the HashSet of tags to include
+        if let Some(host_path) = log.host_path() {
+            self.tags.replace(host_path);
+        }
```

Comment:
If the host is in the metadata it's not going to be in the data we send. I'm thinking it probably should be? Do we want to only add this if the `host_path` is in the main event data?

Reply:
🤔 I was originally thinking it would "just work" but I don't think `convert_to_fields()` accesses the metadata portion... though thinking about it, the behavior is likely what you're suggesting?

We'll check every key at `.`, if it's not there - it doesn't get added.

I think it's more likely we'd want to intentionally pull the `host` from the metadata and ensure it's part of the tag set?

Reply:
Should we test this behavior to confirm?

Reply:
Yes, the current behavior with this change is to ignore anything that's not on the event itself. Any path in the metadata isn't included.

Additionally the encoding fails if the root isn't an object, which is possible with the new log namespacing.